### PR TITLE
Vorticity Example from MHS

### PR DESCRIPTION
**Pull request by jpfairbanks**
This is a discretization of the incompressible Navier Stokes equations using the Discrete Exterior Calculus.

The formulations are based on those given by [Mohamed, Hirani, Samtaney](https://arxiv.org/abs/1508.01166) (in turn from [Marsden, Ratiu, Abraham](https://link.springer.com/book/10.1007/978-1-4612-1029-0)).

However, different choices in discretization are chosen for purposes of brevity, to demonstrate novel discretizations of certain operators, and to demonstrate the automated Decapodes workflow.

### CODE REVIEW

File: docs/src/navier_stokes/ns.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Navier Stokes Vorticity Model
+
+Decapodes can be used to implement
```
Comment (GeorgeR227) [inline docs/src/navier_stokes/ns.md:3]: Remove or expand on this line.

**Review by GeorgeR227 — APPROVED** [review]
Doc builds are still failing but this should be visited in future PR.

**Comment by jpfairbanks** [comment]
I had to comment out Cahn_Hilliard.md to get the docs to build locally. We can remove that page in this PR, and then fix it in another PR.

**Comment by GeorgeR227** [comment]
I think the problem was that a new release of Decapodes hadn't been put out in a while. I think Cahn Hilliard should work on the latest.

**Comment by lukem12345** [comment]
Need to squash for next time

**Pull request merged**